layouts/default.vue

```diff
@@ -99,7 +108,19 @@
     },
 
     watch: {
-      '$i18n.locale': '$fetch'
+      '$i18n.locale': '$fetch',
+      $route(to, from) {
+        this.$nextTick(() => {
+          if (to.path === from.path) {
+            // this.enableAnnouncer = false;
+          } else {
+            this.$refs.resetfocus.setAttribute('tabindex', '0');
+            this.$refs.resetfocus.focus();
+            this.enableAnnouncer = true;
+          }
```

Comment:
if enableAnnouncer isn't needed this can simply be:
```
          if (to.path !== from.path) {
            this.$refs.resetfocus.setAttribute('tabindex', '0');
            this.$refs.resetfocus.focus();
          }
```

Reply:
This was a mistake. We do need to disable the announcer when the path stays the same (to prevent an announcement after each application of a search filter for example.
I commented it out to test something and then forgot to uncomment it. In the last commit I brought it back in.